installers/redhat-docker/radiasoft-download.sh

```diff
@@ -60,6 +50,9 @@ Then re-run this command
         install_err "installer does not support os=$install_os_release_id"
     fi
     install_yum_install docker-ce
+    if [[ ! ${redhat_docker_dev:-} ]]; then
```

Comment:
@robnagler bringing [our conversation](https://github.com/radiasoft/download/pull/708#discussion_r1699085015) back up. I don't think there is a test for the "computing context" that will work. The test we need to know is are we in rsconf or not. Any ideas?

Reply:
I think it's pretty clear that we do not want to execute `usermod -aG docker vagrant` arbitrarily. The case where we want to allow this is a virtual machine that is set up for development. The usermod gives the user root privs without a password so the test to see whether the user vagrant is in sudoers with nopassword is the right one. This will be true in vagrant VMs.

Reply:
I don't think we want it to be true in v3/v4 vms managed by rsconf. There will be unnecessary differences between what we are testing in dev and what happens in higher envs.

For a "v" (or vm_devbox) vm then I agree.

Reply:
We can have the default for the environment variable, and then override it in this case. There are already special things we do to run rsconf in those environments.

Reply:
7c892d9